analysis/aws_rules_cis/aws_cloudtrail_stopped.yml

```diff
@@ -1,7 +1,7 @@
 AnalysisType: rule
-Filename: aws_cloudtrail_modified.py
-PolicyID: AWS.CloudTrail.CloudTrailModified
-DisplayName: CloudTrail Modified
+Filename: aws_cloudtrail_stopped.py
+PolicyID: AWS.CloudTrail.CloudTrailStopped
+DisplayName: CloudTrail Stopped
```

Comment:
I might even say this could be bumped to a high priority issue if it's just for CloudTrail being removed now. (I can't comment directly on the severity portion because it was not part of this change.)